purefa/README.md

```diff
@@ -0,0 +1,123 @@
+# Agent Check: PureFA
+
+## Overview
+
+This check monitors the [Pure Storage FlashArray][1] through the [Datadog Agent][2] and the [Pure Storage Prometheus exporter][1].
```

Comment:
It would also be nice to have a bit more context here about what kind of data this integration displays to the user - i.e. what specific information about their Pure Storage FlashArray will the user have access to in Datadog?

Reply:
Ack